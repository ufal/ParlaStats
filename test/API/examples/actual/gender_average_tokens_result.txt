[{'person.sex': '', 'average_tokens': 1335.3333333333333}, {'person.sex': 'M', 'average_tokens': 325.23359818388195}, {'person.sex': 'F', 'average_tokens': 245.54014029618082}]

+------------+--------------------+
| person.sex |   average_tokens   |
+------------+--------------------+
|            | 1335.3333333333333 |
|     M      | 325.23359818388195 |
|     F      | 245.54014029618082 |
+------------+--------------------+
