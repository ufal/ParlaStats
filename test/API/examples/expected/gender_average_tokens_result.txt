[{'person.sex': '', 'average_tokens': 479.80645161290323}, {'person.sex': 'M', 'average_tokens': 153.0463362811891}, {'person.sex': 'F', 'average_tokens': 168.76321723135348}]

+------------+--------------------+
| person.sex |   average_tokens   |
+------------+--------------------+
|            | 479.80645161290323 |
|     M      | 153.0463362811891  |
|     F      | 168.76321723135348 |
+------------+--------------------+
